mathics/doc/common_doc.py

```diff
@@ -1211,7 +1624,7 @@ def load_documentation_sources(self):
         # Now extract Appendix information. This include License text.  This is
         # the final Part of the documentation.
 
-        for part in appendix:
+        for part in self.appendix:
```

Comment:
In order to decompose the loading in subtasks, I need to put the appendix variable as a class variable.